supabase.types.ts

```diff
@@ -290,6 +320,21 @@ export interface Database {
           }
         ]
       }
+      tags: {
+        Row: {
+          tag_id: string
```

Comment:
given that the `tag_name` field has unique & not null constraints, there is no real need for a separate id to index on